docs/styling/common-props.md

```diff
@@ -0,0 +1,241 @@
+        "values": ["solid", "dashed red", "thick double #32a1ce", "4mm ridge rgba(211, 220, 50, .6)"],
+        "link": "https://developer.mozilla.org/en-US/docs/Web/CSS/border-bottom",
+    },
+    "border_color": {
+        "description": "Sets the color of an element's border (each side can be set individually using border_top_color, border_right_color, border_bottom_color, and border_left_color)",
+        "values": ["red", "red #32a1ce", "red rgba(170, 50, 220, .6) green", "red yellow green transparent"],
+        "link": "https://developer.mozilla.org/en-US/docs/Web/CSS/border-color",
+    },
+    "border_radius": {
+        "description": "Rounds the corners of an element's outer border edge and you can set a single radius to make circular corners, or two radii to make elliptical corners",
+        "values": ["30px", "25% 10%", "10% 30% 50% 70%", "10% / 50%"],
+        "link": "https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius",
+    },
+    "border_width": {
+        "description": "Sets the width of an element's border",
+        "values": ["thick", "1em", "4px 1.25em", "0 4px 8px 12px"],
+        "link": "https://developer.mozilla.org/en-US/docs/Web/CSS/border-width",
+    },
+    "box_shadow": {
+        "description": "Adds shadow effects around an element's frame. You can set multiple effects separated by commas. A box shadow is described by X and Y offsets relative to the element, blur and spread radius, and color",
+        "values": ["10px 5px 5px red", "60px -16px teal", "12px 12px 2px 1px rgba(0, 0, 255, .2)", "3px 3px red, -1em 0 .4em olive;"],
+        "link": "https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow",
+    },
+
+    "color": {
+        "description": "Sets the foreground color value of an element's text",
+        "values": ["rebeccapurple", "rgb(255, 255, 128)", "#00a400"],
+        "link": "https://developer.mozilla.org/en-US/docs/Web/CSS/color",
+    },
+    "display": {
+        "description": "Sets whether an element is treated as a block or inline box and the layout used for its children, such as flow layout, grid or flex",
+        "values": ["block", "inline", "inline-block", "flex", "inline-flex", "grid", "inline-grid", "flow-root"],
+        "link": "https://developer.mozilla.org/en-US/docs/Web/CSS/display",
+    },
+    "flex_grow": {
+        "description": " Sets the flex grow factor, which specifies how much of the flex container's remaining space should be assigned to the flex item's main size",
+        "values": ["1", "2", "3"],
+        "link": "https://developer.mozilla.org/en-US/docs/Web/CSS/flex-grow",
+    },
+    "height": {
+        "description": "Sets an element's height",
+        "values": ["150px", "20em", "75%", "auto"],
+        "link": "https://developer.mozilla.org/en-US/docs/Web/CSS/height",
+    },
+    "justify": {
```

Comment:
similarly here, i wonder if we should list `justify_content`

Reply:
same as above